diff --git a/modules/bar/BarContent.qml b/modules/bar/BarContent.qml
--- a/modules/bar/BarContent.qml
+++ b/modules/bar/BarContent.qml
@@ -403,0 +404,6 @@
+                        YtdButton {
+                            bar: root
+                            layerEnabled: root.shadowsEnabled
+                            startRadius: root.innerRadius
+                            endRadius: root.innerRadius
+                        }
@@ -657,0 +664,6 @@
+                                YtdButton {
+                                    bar: root
+                                    layerEnabled: root.shadowsEnabled
+                                    vertical: true
+                                    startRadius: root.innerRadius
+                                    endRadius: root.innerRadius
+                                }
